src/python/pants/backend/terraform/testutil.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I believe naming a helper, test support module `testutil.py` is fairly common in the repo so I thought it makes sense to use that.

```
$ find . -type f -name "*test*util.py"
./src/python/pants/backend/go/testutil.py
./src/python/pants/backend/helm/testutil.py
./src/python/pants/backend/terraform/testutil.py
./src/python/pants/bsp/testutil.py
./src/python/pants/engine/internals/engine_testutil.py
./src/python/pants/jvm/resolve/coursier_test_util.py
./src/python/pants/jvm/testutil.py
./src/python/pants/testutil/pytest_util.py
./tests/python/pants_test/init/test_util.py
```